stories/forms/inputs/input.story.js

```diff
@@ -15,33 +15,59 @@ const inputProps = {
 
 storiesOf('Input', module)
   .add('with default label', () => (
-    <Input 
-      input={inputProps} 
+    <Input
+      input={inputProps}
       meta={{}}
     />
   ))
   .add('with custom label', () => (
-    <Input 
+    <Input
       input={inputProps}
       meta={{}}
       label="Custom Label"
     />
   ))
   .add('with no label', () => (
-    <Input 
+    <Input
       input={inputProps}
       meta={{}}
       label={false}
     />
   ))
+  .add('with required true default indicator', () => (
+    <Input
+      input={inputProps}
+      meta={{}}
+      label="Custom Label"
+      required={ true }
```

Comment:
If a prop is true you can just write `required` without explicitly setting it to true. You don't have to change this I think it's more of a personal preference thing

Reply:
I keep forgetting, thanks for the reminder!